tests/models/tvlt/test_feature_extraction_tvlt.py

```diff
@@ -0,0 +1,325 @@
+        encoded_pixels = feature_extractor(video_inputs, return_tensors="pt").pixel_values
+        self.assertEqual(
+            encoded_pixels.shape,
+            (
+                self.feature_extract_tester.batch_size,
+                self.feature_extract_tester.num_frames,
+                self.feature_extract_tester.num_channels,
+                self.feature_extract_tester.crop_size["height"],
+                self.feature_extract_tester.crop_size["width"],
+            ),
+        )
+
+    def test_call_pytorch(self):
+        # Initialize feature_extractor
+        feature_extractor = self.feature_extraction_class(**self.feat_extract_dict)
+        # create random PyTorch tensors
+        video_inputs = prepare_video_inputs(self.feature_extract_tester, equal_resolution=False, torchify=True)
+        for video in video_inputs:
+            self.assertIsInstance(video, list)
+            self.assertIsInstance(video[0], torch.Tensor)
+
+        # Test not batched input
+        encoded_pixels = feature_extractor(video_inputs[0], return_tensors="pt").pixel_values
+        self.assertEqual(
+            encoded_pixels.shape,
+            (
+                1,
+                self.feature_extract_tester.num_frames,
+                self.feature_extract_tester.num_channels,
+                self.feature_extract_tester.crop_size["height"],
+                self.feature_extract_tester.crop_size["width"],
+            ),
+        )
+
+        # Test batched
+        encoded_pixels = feature_extractor(video_inputs, return_tensors="pt").pixel_values
+        self.assertEqual(
+            encoded_pixels.shape,
+            (
+                self.feature_extract_tester.batch_size,
+                self.feature_extract_tester.num_frames,
+                self.feature_extract_tester.num_channels,
+                self.feature_extract_tester.crop_size["height"],
+                self.feature_extract_tester.crop_size["width"],
+            ),
+        )
+
+        
+class TVLTAudioFeatureExtractionTester(unittest.TestCase):
+    def __init__(
+        self,
+        parent,
+        batch_size=7,
+        min_seq_length=128,
+        max_seq_length=2048,
+        audio_size=1024,
+        num_channels=1,
+        feature_size=128,
+        sampling_rate=44100,
+        hop_length=512,
+        chunk_length=30,
+    ):
+
+        self.parent = parent
+        self.batch_size = batch_size
+        
+        self.min_seq_length = min_seq_length
+        self.max_seq_length = max_seq_length
+        self.audio_size = audio_size
+        self.num_channels = num_channels
+        self.hop_length = hop_length
+        self.chunk_length = chunk_length
+        self.sampling_rate = sampling_rate
+
+    def prepare_feat_extract_dict(self):
+        return {
+            "audio_size": self.audio_size,
+        }
+
+
+@require_torch
+@require_vision
+class TVLTAudioFeatureExtractionTest(FeatureExtractionSavingTestMixin, unittest.TestCase):
```

Comment:
Since this follows the whisper implementation, you can add all the test from there, and use the `#Copied from`if possible 👍🏻 Integration tests are also a plus